pkg/api/controller.go

```diff
@@ -3006,6 +3011,72 @@ func (c *Controller) GetCurrentUser(w http.ResponseWriter, r *http.Request) {
 	writeResponse(w, http.StatusOK, response)
 }
 
+func (c *Controller) sendResetPasswordEmail(email []string, token string) error {
+	err := c.Emailer.SendEmail(email, token, token, []string{})
+	if err != nil {
+		c.Logger.
+			WithError(err).
+			WithFields(logging.Fields{
+				"email": email,
+			}).Debug("Error sending email")
+		return err
+	}
+	c.Logger.WithFields(logging.Fields{
+		"email": email,
+	}).Info("Successfully sent email")
+	return err
+}
+
+func (c *Controller) PasswordForgot(w http.ResponseWriter, r *http.Request, body PasswordForgotJSONRequestBody) {
```

Comment:
We should log/metric the error in this flow - but not notify the user about anything - as this request is not authenticated. If it is the real user - the user will accept email - this is the way the request is verified.